peps/pep-0738.rst

```diff
@@ -325,7 +336,8 @@ How to Teach This
 The education needs related to this PEP mostly relate to how developers can
 build Python into an Android app and use it at runtime. Automating and
 documenting this will be the responsibility of higher-level tools such as
-BeeWare's `Chaquopy <https://chaquo.com/chaquopy/>`__ and Kivy's `Buildozer
+`Briefcase <https://briefcase.readthedocs.io/en/stable/>`__,
+`Chaquopy <https://chaquo.com/chaquopy/>`__ and `Buildozer
 <https://buildozer.readthedocs.io/en/latest/>`__, rather than CPython itself.
```

Comment:
The earlier JNI reference semi-implies there will be a discussion about JNI in this section; this is sort of implied by the "how to use it at runtime", but it might benefit from being explicit.

Reply:
In addition - even if the bulk of the details are covered by Briefcase/Chaquopy/Buildozer as "how you use this in practice", it would be worth having some explicit documentation of what is involved in integration. It doesn't need to provide a complete solution - a high level "this is the list of problems you'll need to solve that are Android specific" would be specific.

To be clear - I'm not suggesting we describe those problems here; only that part of the final deliverables should be high level documentation of those integration issues, and that this deliverable should be mentioned here as part of the education approach.

Reply:
I agree that there needs to be enough "how to use at runtime" documentation as well as educating developers and users about the need for wheel support by third party libraries.

Reply:
That makes sense. @willingc has already pointed out [this documentation](https://docs.python.org/3/using/windows.html#the-embeddable-package) on the Windows embeddable package; I'll mention in the PEP that we should produce an equivalent for Android.